src/robot/parsing/model.py

```diff
@@ -597,20 +597,33 @@ def __iter__(self):
 
 
 class ForLoop(_WithSteps):
+    """
+    py:class:: ForLoop(declaration, comment=None)
```

Comment:
This isn't needed nor is it parsed anyway separately. I think you would use something like this if you were writing a separate API doc using Sphinx, but we are using `autodoc` extension which can figure out the types of documented items automatically.

A short plain-text description for the class would be nice, though. It should be placed right after the opening quotes to follow PEP-257.